Lib/test/test_minidom.py

```diff
@@ -79,16 +79,16 @@ def testParseFromTextFile(self):
 
     def testAttrModeSetsParamsAsAttrs(self):
         attr = Attr("qName", "namespaceURI", "localName", "prefix")
-        self.assertEqual(attr._name, "qName")
+        self.assertEqual(attr.name, "qName")
         self.assertEqual(attr.namespaceURI, "namespaceURI")
-        self.assertEqual(attr._prefix, "prefix")
-        self.assertEqual(attr._localName, "localName")
+        self.assertEqual(attr.prefix, "prefix")
+        self.assertEqual(attr.localName, "localName")
 
     def testAttrModeSetsNonOptionalAttrs(self):
         attr = Attr("qName", "namespaceURI", None, "prefix")
-        self.assertEqual(attr._name, "qName")
+        self.assertEqual(attr.name, "qName")
         self.assertEqual(attr.namespaceURI, "namespaceURI")
-        self.assertEqual(attr._prefix, "prefix")
+        self.assertEqual(attr.prefix, "prefix")
         self.assertFalse(hasattr(attr, "_localName"))
```

Comment:
Here can you test instead that `attr.localName` falls back to `qName`?

Part of the reason for preferring this approach is that I'd want these tests to remain unchanged if we ever add say a C implementation of minidom. The properties would remain the same, but the private attributes might not.

Reply:
Ah, that makes a lot more sense. Thank you. Apologies for being at all difficult, I really appreciate your willingness to review and provide me with the opportunity to grow and work through feedback, which I don't get as often in other forums.

Thank you again.

Reply:
Fixed in 19fbcc4af1a1105ce27da8935bcd26ed0bb9829e